src/goto-instrument/code_contracts.cpp

```diff
@@ -14,53 +14,107 @@ Date: February 2016
 #include "code_contracts.h"
 
 #include <algorithm>
+#include <cstring>
 #include <map>
+#include <set>
 
 #include <analyses/local_may_alias.h>
 
 #include <ansi-c/c_expr.h>
 
+#include <goto-programs/goto_convert_class.h>
+#include <goto-programs/goto_convert_functions.h>
 #include <goto-programs/remove_skip.h>
 
+#include <ansi-c/expr2c.h>
+#include <linking/static_lifetime_init.h>
 #include <util/arith_tools.h>
 #include <util/c_types.h>
+#include <util/config.h>
 #include <util/expr_util.h>
 #include <util/fresh_symbol.h>
 #include <util/mathematical_expr.h>
 #include <util/mathematical_types.h>
 #include <util/message.h>
 #include <util/pointer_offset_size.h>
 #include <util/pointer_predicates.h>
+#include <util/prefix.h>
 #include <util/replace_symbol.h>
 
-/// Predicate to be used with the exprt::visit() function. The function
-/// found_return_value() will return `true` iff this predicate is called on an
-/// expr that contains `__CPROVER_return_value`.
-class return_value_visitort : public const_expr_visitort
+#include <ansi-c/ansi_c_language.h>
```

Comment:
Please place this line above, together with line 23.

Reply:
Fixed it.